templates/management-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  management:
+    image: yalelibraryit/dc-management:master
+    environment:
+      PASSENGER_APP_ENV: development # development
+      POSTGRES_DB: management_yul_development
+      POSTGRES_HOST: db
+      POSTGRES_PASSWORD: password
+      POSTGRES_USER: postgres
+      SOLR_BASE_URL: http://solr:8983/solr
+      SOLR_CORE: blacklight-core
+      RAILS_RELATIVE_URL_ROOT: /management
+      RAILS_LOG_TO_STDOUT: 'true'
+    ports:
+      - "3001:3001"
```

Comment:
Thought for the future: It would probably help security if at some point we stop mapping the ports outside the containers in production-like environments.

Reply:
small edits:

Thought for the future: It would probably help ~~security~~ everything if at some point we stop mapping the ports outside the containers ~~in production-like environments~~ everywhere.

Reply:
;-)